Cython/Compiler/TypeSlots.py

```diff
@@ -488,7 +488,7 @@ class GetSetSlot(SlotDescriptor):
     #  Slot descriptor for the table of attribute get & set methods.
 
     def slot_code(self, scope):
-        if scope.property_entries:
+        if scope.property_entries or scope.lookup_here("__dict__"):
```

Comment:
This repetitive special casing actually suggests that the feature should be a normal property.